infra-lifecycle-wg/README.md

```diff
@@ -0,0 +1,20 @@
+# Infrastructure Lifecycle WG
+
+The [charter](./charter/charter.md) describes the goals and activites of this group.
+
+To contribute please join our Slack channel and/or biweekly meetings.
+
+#wg-infrastructure-lifecycle Slack channel: <https://cloud-native.slack.com/archives/C06USDTN683>. Get an invite at <https://slack.cncf.io/>
+
+## Chairs
+
+- Bruno Schaatsbergen (@bschaatsbergen)
+- Dylan Page (@genpage)
+- Jochen Zehnder (@elft3r)
+- Ryan Nowak (@rynowak)
```

Comment:
Is there a reason why we need to start with 4 co-chairs?  We'd recommend 2-3 in general as working through a leadership draft doc.

Reply:
This was an active discussion with TAG Chairs @roberthstrand and @lianmakesthings. I have already self-nominated for TAG Tech Lead, intending to step down as chair and step in as the TAG Liaison for the WG should I be selected.

In the interim, the current TAG sponsor, @roberthstrand, will be a tie-breaker if needed.

Reply:
Ok cool, that sounds good, thx for the background info!